facebook/Makefile

```diff
@@ -72,6 +72,7 @@ $(TODAY):
 	[ -f $(QUALTRICS) ] || mkdir -p $(QUALTRICS)
 	rm -rf tmp
 	touch -d "$(TODAY) 01:00" tmp
+	$(PYTHON) -m delphi_utils set end_date $(YESTERDAY)
```

Comment:
Looks like this procedure already appears on line 91, in the `params.json` target. I'm not understanding why this is happening twice.

Reply:
Nvm, got it.